dotnet/src/Connectors/Connectors.Memory.Postgres/PostgresConstants.cs

```diff
@@ -59,7 +59,11 @@ internal static class PostgresConstants
             typeof(Guid),
         ],
 
-        SupportedVectorPropertyTypes = [typeof(ReadOnlyMemory<float>)]
+        SupportedVectorPropertyTypes =
+        [
+            typeof(ReadOnlyMemory<float>),
+            typeof(ReadOnlyMemory<float>?)
```

Comment:
note to self: we need to extend the conformance tests with a nullable ROM property